code/classes/DMSRequestItem.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * Class DMSRequestItem wrapper which represents a DocumentCartItem
+ */
+class DMSRequestItem extends ViewableData
+{
+    /**
+     * The number of copies required of @itemID
+     * @var Int
+     */
+    private $quantity;
+
+    /**
+     * The linked {@link DMSDocument}
```

Comment:
presume this means there's a document linked to an instance of this class - can you please make your doc blocks explanatory for someone like me who doesn't know what your code does?